cmd/simplerequester/README.md

```diff
@@ -0,0 +1,118 @@
+SIMPLE requester
```

Comment:
don't mix markdown headers style, just use a single `#` here

Reply:
lets move SIMPLE requester to another branch and task and PR